experimental/dds/tree/src/EditLog.ts

```diff
@@ -74,6 +74,11 @@ export interface OrderedEditSet<TChange = unknown> {
 	 * @deprecated this will be removed in favor of {@link OrderedEditSet.tryGetEditAtIndex}
 	 */
 	getEditInSessionAtIndex(index: number): Edit<TChange>;
+
+	/**
+	 * @returns all local edits.
```

Comment:
Nit: it is generally preferred to use the standard summary comment component over specific `@<tag>` blocks like `@returns`.
E.g.
```suggestion
	 * Gets all local edits.
```

This improves the generated docs experience. It also has the side effect of removing the `(undocumented)` comment in the API report.